pages/docs/tutorials/native/gradle-for-kotlin-native.md

```diff
@@ -0,0 +1,161 @@
+---
+type: tutorial
+layout: tutorial
+title:  "Gradle for Kotlin/Native"
+description: "Using Gradle when working with Kotlin/Native"
+authors: Hadi Hariri 
+date: 2018-01-24
+showAuthorInfo: false
+---
+
+
+Up to now, most of the tutorials have been using the [command line compiler](basic-kotlin-native-app.html) to build applications, including linking in any `klib` libraries
+either [produced directly](working-with-klib.html) or as a result of [interoperability with C](interop-with-c.html). While it is certainly possible to use the command line, be it directly or
+combine it with a script file (i.e. bash or bat file), it is more recommendable to use a build tool for this process.  
+
+Kotlin/Native has a plugin for [Gradle](https://gradle.org) which allows us to compile, build libraries, reference libraries, and in general perform all the actions that we would need to build our application. 
+
+In this tutorial we'll see how to
+
+* [Build an application with Gradle](#building-an-application)
+* [Build and reference a library with Gradle](#building-and-referencing-a-library)
+* [Generate CMake files to use with CLion](#generating-cmake-files)
+
+
+**Note**: This tutorial assumes somewhat familiarity with Gradle. If you're new to Gradle, while you should be able to follow along, it would be beneficial to review some of the 
+[Getting Started guides](https://gradle.org/guides/#getting-started). To install Gradle on your system please see the [Install guide](https://gradle.org/install/).
+
+
+## Building an Application
+
+The following Gradle script (named `build.gradle`) is the simplest needed to compile an application 
+
+```groovy
+buildscript {
+    repositories {
+        mavenCentral()
+        maven {
+            url "https://dl.bintray.com/jetbrains/kotlin-native-dependencies"
+        }
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-native-gradle-plugin:+"
+    }
+}
+
+apply plugin: 'konan'
+
+konan.targets = ['macbook']
+
+konanArtifacts {
+    program('hello')
+}
+```
+
+The `buildscript` section defines the repositories and dependencies required for the Gradle plugin for Kotlin/Native which is `konan`. Then comes the actual parameters for compiling:
```

Comment:
"then comes ... parameters" is ungrammatical; "then come" would be correct. Or rephrase.